src/vs/workbench/browser/parts/compositePart.ts

```diff
@@ -417,6 +417,14 @@ export abstract class CompositePart<T extends Composite> extends Part {
 		const $this = this;
 		return {
 			updateTitle: (id, title, keybinding) => {
+				// MEMBRANE: hide title bar for Membrane Navigator
+				if (id.includes('membraneContainer')) {
+					parent.style.display = 'none';
+				} else {
+					// reset title bar display when toggling to another composite
+					parent.style.display = 'flex';
+				}
+
```

Comment:
This took me a lot longer to figure out than it should have, but it also caused another issue that I am trying to fix. The parent container is now smaller without the title bar, so there's 35px of dead space at the bottom:
<img width="546" alt="Screenshot 2024-08-07 at 10 51 23 AM" src="https://github.com/user-attachments/assets/4414d740-2415-414d-8df7-0cd1df4b0093">

Reply:
It also flashes a bit while the page is loading.